run_dir/static/js/user_management.js

```diff
@@ -0,0 +1,174 @@
+        //Main table
+        var tbl_row = $('<tr>');
+        user_td = '<div class="btn-group">' +
+                  '<button class="btn btn-sm ml-2 btn-outline-primary btn-large modify-user-btn" data-user=' + name + '><i class="fa fa-wrench mr-1"></i>Modify</button>' +
+                  '<button class="btn btn-sm btn-outline-danger btn-large delete-user-btn'
+        if (name == $('#asrol-js').data('user')){
+          user_td += ' disabled'
+        }
+        user_td += '" data-user=' + name + '><i class="fa fa-times mr-1"></i>Delete</button></div>'
+        user_td += '<span class="ml-3">' + name + '</span>'
+
+        tbl_row.append($('<td>').html(user_td));
+
+        if(role){
+          role_list = Object.values(role).join(', ')
+          tbl_row.append($('<td>').html(role_list))
+        }
+        else{
+          tbl_row.append($('<td>')
+            .html('')
+          );
+        }
+        $("#user_table_body").append(tbl_row);
+      })
+
+      init_listjs();
+
+      $('.modify-user-btn').click(function() {
+          user = $(this).attr('data-user')
+          roles = tableData[user]
+          /* reset the checkboxes */
+          $('.modify-user-role-checkbox input').each(function() {
+              $(this).prop("checked", false);
+          })
+
+          /* Fill in the checkboxed based on the chosen users roles */
+          $.each(roles, function(role, Role) {
+              checkbox_id = '#check_' + role;
+              $(checkbox_id).prop("checked", true);
+          });
+      });
+    })
+ }
+
+ function init_listjs() {
+      // Setup - add a text input to each footer cell
+      $('#ur_table tfoot th').each( function () {
+        var title = $('#ur_table thead th').eq( $(this).index() ).text();
+        $(this).html( '<input size=10 type="text" placeholder="Search '+title+'" />' );
+      } );
+
+      var table = $('#ur_table').DataTable({
+        "paging":false,
+        "info":false,
+        "order": [[ 0, "asc" ]]
+      });
+
+      //Add the bootstrap classes to the search thingy
+      $('div.dataTables_filter input').addClass('form-control search search-query');
+      $('#ur_table_filter').addClass('form-inline float-right mt-2');
+      $("#ur_table_filter").appendTo("#searchGoesHere");
+      $('#ur_table_filter label input').appendTo($('#ur_table_filter'));
+      $('#ur_table_filter label').remove();
+      $("#ur_table_filter input").attr("placeholder", "Search..");
+      // Apply the search
+      table.columns().every( function () {
+          var that = this;
+          $( 'input', this.footer() ).on( 'keyup change', function () {
+              that
+              .search( this.value )
+              .draw();
+          } );
+      } );
+  }
+
+  $('body').on('click', '#submitCreateUserBtn', function(event){
+    var roles={};
+    $('.create-user-role-checkbox input:checked').each(function() {
+        role = this.value
+        Role = $(this).attr('data-label')
```

Comment:
I'm not really sure about the variable naming here. Should we begin with a capital letter?